Assets/Scripts/UIScripts/MainMenu.cs

```diff
@@ -1,19 +1,53 @@
+using System;
 using System.Collections;
 using System.Collections.Generic;
+using TMPro;
 using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.DualShock;
+using UnityEngine.InputSystem.Switch;
+using UnityEngine.InputSystem.XInput;
 using UnityEngine.SceneManagement;
 
 public class MainMenu : MonoBehaviour
 {
     public AudioSource musicIntro; 
     public AudioSource musicLoop;
+    public GameObject ps4Prompt;
+    public GameObject xboxPrompt;
+    private Gamepad _gamepad;
 
     public void Start()
     {
         musicIntro.Play();
         musicLoop.PlayDelayed(musicIntro.clip.length);
+        
+        _gamepad = Gamepad.current;
+
+        if (_gamepad != null)
+        {
+            Cursor.lockState = CursorLockMode.Locked;
+            Cursor.visible = false;
+        }
     }
-    
+
+    public void Update()
```

Comment:
This is good, but I think it could be better if we instead use an event that triggers when a new controller joins or input from a different gamepad is detected. We're starting to have a lot of these Update() methods which is going to cause performance issues. I looked online and found an overriden method in the InputSystem called MakeCurrent() that triggers when there is input on the gamepad: https://docs.unity3d.com/Packages/com.unity.inputsystem@1.4/api/UnityEngine.InputSystem.Gamepad.html#:~:text=%22axis%22)%3B%0A%20%20%20%20%7D%0A%7D-,MakeCurrent(),-Make%20the%20gamepad
You can probably just add your code to that method and it should work

Actually, I just realized this is in the MainMenu scene which doesn't have any Update() methods and also is completely separate from the level one scene oops...I guess you could leave it for now and create a polish task?

Reply:
Yup I was wondering whether I should have this in Update() or not, but then I realized this is only for the Main Menu screen, so it will not be running for too long.

I'll add a polish task 👍🏼